src/rubrix/client/api.py

```diff
@@ -213,15 +213,12 @@ def copy(self, dataset: str, name_of_copy: str, workspace: str = None):
             >>> rb.copy("my_dataset", name_of_copy="new_dataset")
             >>> rb.load("new_dataset")
         """
-        response = datasets_api.copy_dataset(
+        datasets_api.copy_dataset(
```

Comment:
Nice catch!!

Reply:
It popped up as unreached code in the coverage review, so I went to check if there was a test for the case where the dataset already exists. I knew something was off when that test case existed, but the code here still wasn't being reached 😉